usr/include/slotmgr.h

```diff
@@ -30,10 +30,10 @@
 #define TOK_PATH  SBIN_PATH "/pkcsslotd"
 #define OCK_API_LOCK_FILE LOCKDIR_PATH "/LCK..APIlock"
 
-#define PROC_SOCKET_FILE_PATH "/var/run/pkcsslotd.socket"
-#define ADMIN_SOCKET_FILE_PATH "/var/run/pkcsslotd.admin.socket"
+#define PROC_SOCKET_FILE_PATH "/run/pkcsslotd.socket"
+#define ADMIN_SOCKET_FILE_PATH "/run/pkcsslotd.admin.socket"
```

Comment:
From FHS: `Programs may have a subdirectory of /run; this is encouraged for programs that use more than one run-time file.`  Should we add a subdirectory for the sockets?

Reply:
Good point. I have changed the commit, please check.